randovania/games/dread/exporter/patch_data_factory.py

```diff
@@ -267,7 +268,7 @@ def _encode_hints(self) -> list[dict]:
             self.description.all_patches,
             self.players_config,
             self.rng,
-            ["A joke hint."],
+            JOKE_HINTS,
```

Comment:
Dunc said that it is possible for joke hint text to appear in CS, so it should get a changelog entry.

Reply:
meh. I don't think anyone has ever used that setting anyway

Reply:
changelog entry is for listing public changes, regardless of whether people use it :p

Reply:
I would specifically prefer it not be listed here, since there are pretty good chances I remove the option altogether